job-server/src/spark.jobserver/JobStatusActor.scala

```diff
@@ -32,6 +33,13 @@ class JobStatusActor(jobDao: JobDAO) extends InstrumentedActor with YammerMetric
   val metricNumJobInfos = gauge("num-running-jobs", infos.size)
   val metricStatusRates = mutable.HashMap.empty[String, Meter]
 
+  override def postStop(): Unit = {
+    val stopTime = DateTime.now()
+    val stoppedInfos = infos.values.map { info =>
+      info.copy(endTime = Some(stopTime), error = Some(new Exception("Context for this job was terminated")))}
```

Comment:
Should we add more info for the user, like:
```scala
 Some(new Exception(s"Context (${info.contextName}) for this job (${info.jobId}}) was terminated")))}
```

I think the job Id might be redundant